strum_macros/src/macros/enum_is.rs

```diff
@@ -0,0 +1,60 @@
+use crate::helpers::non_enum_error;
+use convert_case::{Case, Casing};
+use proc_macro2::TokenStream;
+use quote::{format_ident, quote};
+use syn::{Data, DeriveInput};
+
+pub fn enum_is_inner(ast: &DeriveInput) -> syn::Result<TokenStream> {
+    let variants = match &ast.data {
+        Data::Enum(v) => &v.variants,
+        _ => return Err(non_enum_error()),
+    };
+
+    let enum_name = &ast.ident;
+
+    let variants: Vec<_> = variants
```

Comment:
strum supports a `disabled` attribute on enums. Should we support that for this derive as well?

Reply:
Don't see any docs for disabled. 

https://docs.rs/strum/latest/strum/?search=disabled returns no results.

Reply:
See [this section](https://docs.rs/strum/latest/strum/additional_attributes/index.html#attributes-on-variants)

Reply:
Added disabled support